tests/child_process/host/host.cpp

```diff
@@ -101,6 +102,35 @@ int main(int argc, const char* argv[])
                 exit(1);
             }
             break;
+        case CREATE_IN_CHILD_PROCESS:
+            if (pid == 0) // child process
+            {
+                fprintf(stdout, "child pid = %d\n", getpid());
+                oe_enclave_t* enclave_in_child_process = NULL;
+                result = oe_create_child_process_enclave(
+                    argv[1],
+                    OE_ENCLAVE_TYPE_SGX,
+                    flags,
+                    NULL,
+                    0,
+                    &enclave_in_child_process);
+                OE_TEST(result == OE_OK);
+                oe_terminate_enclave(enclave_in_child_process);
+                _exit(EXIT_SUCCESS);
+            }
+            else if (pid > 0) // parent process
+            {
+                fprintf(stdout, "parent pid = %d\n", getpid());
+                sleep(5);
```

Comment:
Can you also add comment about the need of `sleep` here?

Reply:
There used to be an issue in creating enclave in child process occasionally. It was fixed, so it doesn't need to repeat creating again and again any more. I can remove the sleeping in the parent process since it is really unnecessary.

Reply:
Thanks!

Reply:
It would be great if you can squash the comments into a single comment.

Reply:
Done.

Reply:
Thanks so much!